korge/src/commonMain/kotlin/com/soywiz/korge/view/filter/Convolute3Filter.kt

```diff
@@ -21,37 +21,37 @@ class Convolute3Filter(
 
         /** A Gaussian Blur Kernel. This [Matrix3D] can be used as [kernel] for [Convolute3Filter] */
         val KERNEL_GAUSSIAN_BLUR: Matrix3D = Matrix3D.fromRows3x3(
-            1f, 2f, 1f,
-            2f, 4f, 2f,
-            1f, 2f, 1f
-        ) * (1f / 16f)
+            1.0, 2.0, 1.0,
+            2.0, 4.0, 2.0,
+            1.0, 2.0, 1.0
+        ) * (1.0 / 16.0)
```

Comment:
This is incidentally wrong. Matrix3D uses FloatArray as backing array.
I think we should update KorMA to support function calling using Floats in addition to doubles here: https://github.com/korlibs/korma/blob/master/korma/src/commonMain/kotlin/com/soywiz/korma/geom/Matrix3D.kt

Reply:
Ok, I update the PR later

Reply:
https://github.com/korlibs/korma/pull/31